src/main/java/com/minecolonies/coremod/colony/managers/IStatisticAchievementManager.java

```diff
@@ -1,5 +1,6 @@
 package com.minecolonies.coremod.colony.managers;
 
+import com.minecolonies.coremod.achievements.MineColoniesAchievement;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.achievements.MineColoniesAchievement'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)